early-return/workflow.go

```diff
@@ -0,0 +1,82 @@
+package earlyreturn
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	UpdateName    = "early-return"
+	TaskQueueName = "early-return-tq"
+)
+
+var (
+	activityTimeout    = 2 * time.Second
```

Comment:
I understand we want to demonstrate low latency, but this is a pretty aggressive timeout. But it is probably ok.

Reply:
Yeah, the idea for the sample is to push the low latency story; and this is actually on the upper end of what I'm hearing for customer use cases. 100-500ms for the entire Update is what's often needed.

Reply:
If that's the case, may not want a default retry policy with an initial interval of a second. May actually want max attempts as 1.

Reply:
I would be as lax as you can on these timeouts. Don't want to unnecessarily fail requests that would have otherwise succeeded. So if the overall workflow task timeout is 10s, maybe give it 9 or 10s?